Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -259,6 +314,12 @@ props.buildReportOrder.each { buildReportFile ->
 					record.getOutputs().each {println("   ${it.dataset}, ${it.deployType}")}
 				}
 			}
+
+			// Log deleted files
+        	if (deletionCount != 0) {
```

Comment:
I would have put this block outside of the condition at line 295, and keep it in its own condition after.